model/shared/src/main/scala/explore/model/SpectroscopyConfigurationOptions.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.Eq
+import eu.timepit.refined.types.numeric.PosBigDecimal
+import eu.timepit.refined.types.numeric.PosInt
+import eu.timepit.refined.cats._
+import explore.model.enum.FocalPlaneOptions
+import explore.model.enum.SpectroscopyCapabilities
+import lucuma.core.math.Angle
+import lucuma.core.math.Wavelength
+import monocle.macros.Lenses
+
+@Lenses
+final case class SpectroscopyConfigurationOptions(
```

Comment:
We will probably switch to using generated classes once there is a model to generate classes from.  But in the meanwhile we do need this.

Reply:
Indeed, and it wouldn't be stored in the state but rather come from a query